test/postgres.js

```diff
@@ -552,6 +552,54 @@ describe('bookshelf-jsonapi-params with postgresql', () => {
                     done();
                 });
         });
+
+    });
+
+    describe('passing in "or" filtering', () => {
+        it('should return results for "or" filters combination', (done) => {
```

Comment:
Can you add a few tests for combing base filters with or filters?

Reply:
And some json filtering inside of `or`

Reply:
@alechirsch Added few test for filter combinations. Just thought that actually every filter it's a combination of "or" filters with one ("base filter") or more ("or filter") filter fragments. So every filter can be described as array of filter fragments